infra-templates/network-diagnostics/1/rancher-compose.yml

```diff
@@ -1,5 +1,19 @@
 .catalog:
-  version: 0.0.1
+  version: 0.0.2
+  minimum_rancher_version: v1.6.5-rc1
+  services:
+  network-diagnostics:
```

Comment:
ah that explains why it was not working. I am still testing this. so it's in two places. Will fix it.